src/sentry/static/sentry/app/components/createAlertButton.tsx

```diff
@@ -337,24 +337,18 @@ const CreateAlertButton = withApi(
       showPermissionGuide,
       ...buttonProps
     }: Props) => {
+      const hasWizard = organization.features.includes('alert-wizard');
+      const createAlertUrl = (providedProj: string) => {
+        const alertsBaseUrl = `/organizations/${organization.slug}/alerts/${providedProj}`;
+        return `${alertsBaseUrl}/${hasWizard ? 'wizard' : 'new'}/${
+          referrer ? `?referrer=${referrer}` : ''
+        }`;
```

Comment:
nit but this would look cleaner probably as 

```tsx
        const alertsBaseUrl = `/organizations/${organization.slug}/alerts/${providedProj}`;
		const subUrl = hasWizard ? 'wizard' : 'new'

        return `${alertsBaseUrl}/${subUrl}/${referrer ? `?referrer=${referrer}` : ''}`;
```

(removing the weird wrapping of the template string)

Reply:
oooh yea I agree this looks better